app/views/schools/show.html.erb

```diff
@@ -119,3 +119,12 @@
     </div>
   </main>
 </div>
+
+<script>
+  window.addEventListener('load', function(e) {
+    // attach to the accordion container so events bubbling up from sections will be caught
+    document.getElementById('accordion-dfe-signin-help').addEventListener('accordionToggle', function(e) {
```

Comment:
Is it useful to monitor this event globally? In the previous revision all expansions of the accordion were getting logged?

Could the event handler be moved into `application.js`? Or does that not have access to `document` because of webpack?

Reply:
Could this just be `document.getEventListener` in application js - it will only trigger for the custom event, its not like monitoring a generic event such as `click`